api-reference/general-tools/assessment/component-info/component.yml

```
@@ -79,99 +34,364 @@ paths:
       responses:
         '200':
           description: OK
-          headers:
-            Date:
-              schema:
-                type: string
-                example: Mon, 12 Aug 2024 07:41:05 GMT
-            Server:
-              schema:
-                type: number
-                example: WSGIServer/0.2 CPython/3.9.17
-            Content-Type:
-              schema:
-                type: string
-                example: application/json
-            Vary:
-              schema:
-                type: string
-                example: Accept, Cookie, origin
-            Allow:
-              schema:
-                type: string
-                example: POST, OPTIONS
-            X-Frame-Options:
-              schema:
-                type: string
-                example: DENY
-            Content-Length:
-              schema:
-                type: integer
-                example: '51'
-            X-Content-Type-Options:
-              schema:
-                type: string
-                example: nosniff
-            Referrer-Policy:
-              schema:
-                type: string
-                example: same-origin
-            Cross-Origin-Opener-Policy:
+          content:
+            application/json:
               schema:
-                type: string
-                example: same-origin
+                type: object
+              example:
+                success: true
+                data:
+                  autosuggest:
+                    question:
+                      "$schema": http://json-schema.org/draft-07/schema#
```

```suggestion
                      schema:
```

this should work right?

---

It's fine.